tests/unit/OfficeAddresses.spec.ts

```diff
@@ -957,7 +957,7 @@ describe('actions and events', () => {
 
     // verify initial events
     const haveChanges = wrapper.emitted('haveChanges')
-    expect(haveChanges.length).toBe(1)
+    expect(haveChanges.length).toBe(2)
```

Comment:
Why are there 2 initial events -- what are they?

Reply:
Mounted lifecycle and the Watcher

Reply:
If the watcher fires when `getOfficeAddresses` changes, why do you need to call it from mounted() ?

Reply:
(Not that I've never done it before, but it's a code smell when a component emits the same event multiple times on init / when nothing has happened yet.)

Reply:
I agree, see the comment above 😁 I can elaborate more though, if need be,

Reply:
Yeah, just to give my 2 cents: maybe checking in the watch if the value has really changed may resolve it if it's because of the mount that triggers it. I'm not sure but there may be any option to discard it.

Reply:
With the inclusion of `immediate` the changes still stand as it will call the method an additional time.
When the component is mounted and when the watcher is triggered, ie any changes to the office addresses.

Reply:
A small comment where the component emits the event, to describe the init sequence (and thus why there are multiple events), would be great! ❤️

Reply:
Will do 👍